piker/data/_web_bs.py

```diff
@@ -219,18 +221,26 @@ async def recv_task():
             field, then sets the event to wakeup original sender tasks.
             '''
             async for msg in ws:
-                msg = dtype(**msg)
+                try:
```

Comment:
Yeah, so as mentioned above consider maybe trying out something like:

```python
match msg:
    case {
        'method': meth,
        'params': params,
     }:
         # ignore or log or wtv
```

you might be able to use your parsed object form as well but i think you might need a type hierarchy to make it work in `JSONRPCMsg(**msgdict)` style?